regions/shapes/circle.py

```diff
@@ -74,9 +75,9 @@ def contains(self, pixcoord):
         pixcoord = PixCoord._validate(pixcoord, name='pixcoord')
         in_circle = self.center.separation(pixcoord) < self.radius
         if self.meta.get('include', False):
```

Comment:
I think the default value of `include` is `True`. We should change that.

Reply:
for all regions

Reply:
Yes... the default value of `include` is `True`, which means `self.meta.get('include', False)` is probably redundant?  But otherwise, I don't understand what you're saying.  What change are you suggesting?

Reply:
Oh, right, nevermind - if `include` is not specified, it should be `True`.  Got it.